pontoon/base/views.py

```diff
@@ -56,7 +56,15 @@
 
 def home(request):
     """Home view."""
+
+    user = request.user
     project = Project.objects.get(id=1)
+
+    # Redirect user to the selected home page or '/'. Because of circular redirects at home() this logic is moved to
+    # a separate view.
+    if user.is_authenticated() and user.profile.custom_homepage:
```

Comment:
How 'bout this - if the custom_homepage is not set, we [compute](https://irccloud.mozilla.com/pastebin/HthfU0wU/) it and set it?